python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -275,6 +275,17 @@ def project_scaffold_command(
             lexer="bash",
             padding=1,
         ),
+        "Note: DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is an environment variable. If using Microsoft "
+        "Powershell, set it before running 'dagster dev' using this syntax:",
```

Comment:
We shouldn't show both of these messages to the user since it could add some confusion. We should just determine what shell they're using something like https://github.com/sarugaku/shellingham and then post the corresponding message.

Reply:
Happy to implement it then. Could you revert the changes to `app.py`?